sdk/src/main/java/io/opentelemetry/sdk/trace/TracerSdk.java

```diff
@@ -32,11 +32,13 @@
 public final class TracerSdk implements Tracer {
   private static final BinaryFormat<SpanContext> BINARY_FORMAT = new BinaryTraceContext();
   private static final HttpTextFormat<SpanContext> HTTP_TEXT_FORMAT = new HttpTraceContext();
-  private final TracerSharedState sharedState;
+  private final TracerSharedState meterProviderSharedState;
```

Comment:
this doesn't seem right. Did you over-reach on the renaming of the variables?

Reply:
fixed, sorry my `sed` skills failed.